contracts/example/src/contract.rs

```diff
@@ -84,4 +136,48 @@ impl Example {
             &message,
         );
     }
+
+    pub fn send_token(
+        env: Env,
```

Comment:
```suggestion
        env: &Env,
```
we're using `&Env` everywhere for simplicity